README.md

```diff
@@ -21,6 +22,7 @@
 * [Upgrade Guides](#upgrade-guides)
 
 ## Features
+- [x] Asset Features
```

Comment:
Again , asset feature is not really a feature , remove it here as well. Otherwise it can misguide the developers